packages/SidePanel/readme.md

```diff
@@ -0,0 +1,59 @@
+# SidePanel
+
+## Props:
+
+```js
+const propTypes = {
```

Comment:
Duplicating the props here in the `README.md` seems like too much work... although a quick listing might be handy, with a link to the source code, [like this](https://github.com/acl-services/paprika/blob/master/packages/Button/README.md)? I'm not so sure yet what the best approach will be.

Reply:
npm provide a direct link from the npm landing page of the component, as long we use the url property under repository on the package.json file.